docs/source/sections/webapp/releases.rst

```diff
@@ -2,12 +2,45 @@
 Web-App Release Notes
 *********************
 
+* **February 1, 2018** 
+
+ **Please Note:** Work is underway to rework our cohort creation page to better differentiate between samples which are from image data vs. those which are not.
+ 
+ **Issues resolved in Sprint 23 as of 02/01/2018**
+ 
+ New Enhancements
+  - We have enabled DNA methylation data to be used when plotting with genomic build hg38.
+  - The cohort view files page has been updated to File Browser. The File Browser page also now has new filters data level, data type, and disease code. A time stamp has also been added to the CSV file that can be downloaded.
+  - The IGV browser and caMicroscope are now more clearly defined and separated on the File Browser page. 1931
+  - When uploading a set of barcodes to create a cohort the error message has been redefined to be more general.
```

Comment:
I'd avoid saying the error message is general (since this can mean 'less helpful'), and just indicate it was reworded to point the user back at the instructions.